doc/whats_new/v0.21.rst

```diff
@@ -183,6 +183,10 @@ Support for Python 3.4 and below has been officially dropped.
   the gradients would be incorrectly computed in multiclass classification
   problems. :issue:`12715` by :user:`Nicolas Hug<NicolasHug>`.
 
+- |Fix| Early stopping is now checked on a stratified split for
```

Comment:
This is unclear. Perhaps "Fixed a bug where validation sets for early stopping in XxClassifier were not sampled with stratification." ??